.circleci/config.yml

```diff
@@ -256,10 +256,7 @@ jobs:
 
       - run:
           name: Install geckodriver
-          command: |
-            curl --silent --show-error --location --fail --retry 3 --output /tmp/geckodriver_linux64.tar.gz https://github.com/mozilla/geckodriver/releases/download/v0.20.1/geckodriver-v0.20.1-linux64.tar.gz
-            tar xvzf /tmp/geckodriver_linux64.tar.gz -C /usr/local/bin
-            rm /tmp/geckodriver_linux64.tar.gz
```

Comment:
This is just a somewhat unrelated change. While running this locally I found that I had somehow uninstalled geckodriver since I last run this suite. Extracting these instructions to a script makes it easier to install locally.